source/MaterialXView/Viewer.cpp

```diff
@@ -1214,6 +1214,8 @@ void Viewer::loadDocument(const mx::FilePath& filename, mx::DocumentPtr librarie
 {
     // Set up read options.
     mx::XmlReadOptions readOptions;
+    readOptions.readComments = true;
```

Comment:
I think it's useful to always turn these on to try and always preserve the original content.
Maybe also do this for the Graph Editor?